### PR TITLE
[Term Entry] PHP Array term array_combine()

### PR DESCRIPTION
**Pull request by brandon-kong**
### Description

- Added PHP Entry
- Closes issue #2555 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-combine/array-combine.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+Title: '.array_combine()'
+Description: 'The array_combine() function returns an array by using the elements of one "keys" array as keys and the elements of another "values" array as the corresponding values.'
```
Comment (dakshdeepHERE) [inline content/php/concepts/arrays/terms/array-combine/array-combine.md:3]:
```suggestion
Description: 'Returns a new array by combining two arrays, where one array represents keys and the other array represents values.'
```
Let's keep this short and descriptive we can explain it more down below

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
One change from my side

File: content/php/concepts/arrays/terms/array-combine/array-combine.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `array_combine()` returns an array by using the elements of one "keys" array as keys and the elements of another "values" array as the corresponding values.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-combine/array-combine.md:16]:
```suggestion
The **`array_combine()`** returns an array by using the elements of one "keys" array as keys and the elements of another "values" array as the corresponding values.
```

File: content/php/concepts/arrays/terms/array-combine/array-combine.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+## Syntax
+
+```pseudo
+array_combine(array $keys, array $values): array
+```
+
+If the two arrays do not have an equal number of elements, `array_combine()` will throw a `ValueError` as of PHP 8.0.0.
+
+## Example
+
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-combine/array-combine.md:27]: Have a sentence before the example block summarizing what it does.

File: content/php/concepts/arrays/terms/array-combine/array-combine.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+Array
+(
+    [one] => 1
+    [two] => 2
+    [three] => 3
+    [four] => 4
+)
+```
+
+Additionally, if there are multiple keys in the `keys` array, the last value that corresponds to the last occurrence of the key will prevail.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-combine/array-combine.md:49]: Swap this line with the header on line 51.

File: content/php/concepts/arrays/terms/array-combine/array-combine.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    [one] => 1
+    [two] => 3
+    [four] => 4
+)
+```
+
+In the example above, the last value of the key `two` is `3`, so the value `3` is used in the combined array.
+
+## Codebyte Example
+
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-combine/array-combine.md:76]: Have a sentence introducing the codebyte and summarizing what it's doing.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@brandon-kong I did a first maintainer review with a few comments to address in addition to @dakshdeepHERE's comment. You also modified the `yarn.lock` file, which shouldn't have happened. I'll try and fix that for you.

**Comment by SSwiniarski** [comment]
@brandon-kong I fixed the edits to the `yarn.lock` file.

File: content/php/concepts/arrays/terms/array-combine/array-combine.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+```pseudo
+array_combine(array $keys, array $values): array
+```
+
+If the two arrays do not have an equal number of elements, `array_combine()` will throw a `ValueError` as of PHP 8.0.0.
+
+## Example
+
+In this example, we will create an array with the keys `one`, `two`, `three`, and `four` and the values `1`, `2`, `3`, and `4`, respectively.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-combine/array-combine.md:28]:
```suggestion
In this example, an array is created with the keys `one`, `two`, `three`, and `four` and the values `1`, `2`, `3`, and `4`, respectively.
```
We avoid first- and second-person pronouns in Docs.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@brandon-kong one more comment.

**Review by SSwiniarski — APPROVED** [review]
@brandon-kong Looks good for a second review.

File: content/php/concepts/arrays/terms/array-combine/array-combine.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+## Codebyte Example
+
+The following codebyte example creates an array of colors and an array of food. It then combines the two arrays into a new array called `$colorHex` with the colors as keys and the food as values.
+
+```codebyte/php
+<?php
+  $colors = array("red", "green", "blue");
+  $food = array("apple", "broccoli", "blueberry");
+  $colorHex = array_combine($colors, $hex);
+  print_r($colorHex);
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-combine/array-combine.md:84]:
```suggestion
  $colors = array("red", "green", "blue");
  $food = array("apple", "broccoli", "blueberry");
  $colorAndFood = array_combine($colors, $food);
  print_r($colorAndFood);
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@brandon-kong I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by brandon-kong** [comment]
@caupolicandiaz Thanks for your initial review. I have went ahead and made changes according to your suggestions. Thanks in advance.

**Review by caupolicandiaz — APPROVED** [review]
@brandon-kong @SSwiniarski Looks good to merge

**Pull request merged**